scripts/variantstore/wdl/extract/extract_subpop.py

```diff
@@ -7,6 +7,7 @@
  "eas",
  "eur",
  "mid",
+ "nfr",
```

Comment:
I thought this was PR was about changing nfr to nfe (or at least that's what the ticket says)